src/hooks/use-render.ts

```diff
@@ -46,11 +52,13 @@ export default function useRender({ flags, sense }: IEnv) {
         listboxOptions: options.listboxOptions ?? {},
         resources: resourcesArr,
         onFullscreen: options.zoomSelf,
+        isZoomed: options.isZoomed,
         constraints,
+        t: translator,
       },
     );
     return (() => {
       teardown(root);
     });
-  }, [resourcesArr, constraints]);
+  }, [resourcesArr, constraints, options.isZoomed]);
```

Comment:
Do we need to account for `options == undefined`. Maybe not.

Reply:
`options` should be an empty object when no options are passed in.